tests/test_app.py

```diff
@@ -0,0 +1,210 @@
+import boto3
+import mock
+import pytest
+import requests
+import yaml
+
+from botocore.stub import Stubber
+from datetime import datetime
+from dateutil.tz import tzutc
+from freezegun import freeze_time
+from io import StringIO
+
+from app import (
+    create_hostedgraphite_base_metrics,
+    format_cloudwatch_metric_datapoint_for_hostedgraphite,
+    format_config_metric_entry_for_hostedgraphite_base_metric,
+    get_config,
+    get_metric_from_cloudwatch,
+    send_to_hostedgraphite
+)
+
+from .fixtures import config
+
+
+class TestApp:
+
+    def setup_method(self):
+        # Mock out boto3 cloudwatch client
+        self.client = boto3.client('cloudwatch', region_name="eu-west-1")
+        stubber = Stubber(self.client)
+        stubber.activate()
+
+        # Mock out send_to_hostedgraphite method
+        self.send_to_hostedgraphite_mock = mock.patch('app.send_to_hostedgraphite')
+        self.send_to_hostedgraphite_mock.start()
```

Comment:
Should we have an equivalent `.stop()` in a `teardown_method()`?

Reply:
Good shout 👍 Done